owned-manifests/machine-api-controllers.yaml

```diff
@@ -36,18 +36,18 @@ spec:
         key: node.alpha.kubernetes.io/unreachable
         operator: Exists
       containers:
-      - name: controller-manager
+      - name: machineset-controller-manager
```

Comment:
Huzzah. Though I still question the `-manager` bit. Kind of vacuous really. If it's a controller, what does the manager bit mean?

Reply:
These are actually controller managers that can hold 1..N controllers

Reply:
> These are actually controller managers that can hold 1..N controllers

Given what you just wrote shouldn't this remain as 'controller-manager'.

Reply:
`machineset`? That's very confusing. The `manager` binary runs more than the machineset controller. It also runs the machinedeployment controller and node controller. It does not make sense to reduce just to machineset.